doc/spec/flowchart.rst

```diff
@@ -5,4 +5,163 @@ Library Flow Chart
+        penalize them to discover the features that are most correlated with treatment effect heterogeneity. 
+
+    INSTRUMENT
+        Some estimators identify the causal effect of a treatment by considering only a subset of the
+        variation in treatment intensity that is conditionally random given other data features. This
+        subset of the variation is driven by an instrument, which is often some kind of randomization (i.e.
+        an earlier experiment or a lottery). See the Instrumental Variable Regression section for more
+        information on picking a good instrument. 
+
+    LINEAR TREATMENT EFFECTS
+        Some estimators impose the assumption that the outcome is a linear function of the treatment. These
+        estimators can also estimate a non-linear relationship between a treatment and the outcome if the
+        structure of the relationship is known and additively separable (for example, the linear function
+        could include both treatment and treatment-squared for continuous treatments). These linear functions
+        can also include specified interactions between treatments. However, these estimators cannot estimate
+        a fully flexible non-parametric relationship between treatments and the outcome (for example, the
+        relationship cannot be modeled by a forest). 
+
+    LINEAR HETEROGENEITY
+        The CATE function determines how the size of a user’s response to the treatment varies by user
+        features. Some estimators impose the assumption that effect size is a linear function of user features.  
+
+    CONFIDENCE INTERVALS/MODEL SELECTION
+        The MetaLearner and DRLearner estimators offer the choice of any ML estimation model in all stages
+        and allows for model selection via cross validation. This enhances flexibility, but because the
+        sample data is used to choose among models it is impossible to calculate honest analytic confidence
+        intervals. Moreover, most ML estimation approaches introduce bias for regularization purposes, so as
+        to optimally balance bias and variance. Hence, confidence intervals based on such biased estimates
+        will be invalid. For these models it is still possible to construct bootstrap confidence intervals,
+        but this process is slow, may not be accurate in small samples and these intervals only capture the 
+        variance but not the bias of the model. 
+
++---------------------------------------------+----------------+---------------------+--------------------------+------------------+----------------------+-------------------+---------------------+-------------------------- +
+| Estimator                                   | Treatment Type | Requires Instrument | Delivers Conf. Intervals | Linear Treatment | Linear Heterogeneity | Mulitple Outcomes | Multiple Treatments | High-Dimensional Features |
```

Comment:
I think because of the long glossary right above, I would make this table a separate rst file with its own heading on the left in the navigation so that it is more visible.

Reply:
I've split the table out to its own "comparison" page